src/app/receipt/page.tsx

```diff
@@ -0,0 +1,190 @@
+"use client";
+
+import React, { useEffect, useState } from "react";
+import { useSearchParams } from "next/navigation";
+import { usePayementsOrdersHistoryRetrieve } from "@/services/ecommerce/payments/hooks";
+import {
+  Button,
```

Comment:
Noting for later - we should prefer the components in [smoot-design](https://github.com/mitodl/smoot-design) over using the MUI ones directly. (At this stage for this PR in particular it's not so important, since we'll have to rework a bunch of visual stuff once the design for this is ready, so it's fine if this stays as is for now.)